example/timer/odp_timer_accuracy.c

```diff
@@ -0,0 +1,461 @@
+/* Copyright (c) 2018, Linaro Limited
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <inttypes.h>
+#include <stdlib.h>
+
+#include <unistd.h>
+#include <getopt.h>
+
+#include <odp_api.h>
+
+#define START_OFFSET_NS ((uint64_t)(300 * ODP_TIME_MSEC_IN_NS))
+
+typedef struct test_global_t {
+	struct {
+		long int period_ns;
+		long int res_ns;
```

Comment:
to be consistent to period values bellow uint64_t and strtoul() might be better here.

Reply:
Changed to unsigned long long int and strtoull(). uint64_t != ul, e.g. in a 32bit system ul is 32 bits.